services/ui-src/src/verbiage/forms/adminBannerForm.ts

```diff
@@ -63,46 +63,14 @@ export const form = {
         hint: "mm/dd/yyyy (12:00:00am)",
```

Comment:
Two notes on the props for this component: 
1. We won't need to pass through a `hint` anymore.
2. To get the date picker to show up, we need to pass through a from prop and a to prop, per the docs:

>To enable the date picker feature, you must pass both a from<TIME> property (i.e., fromDate, fromMonth, etc.) and a to<TIME> property (i.e., toDate, toMonth, etc.) to the date field component.